packages/engine-core/src/library/LibraryEngine.ts

```diff
@@ -222,6 +225,10 @@ You may have to run ${chalk.greenBright('prisma generate')} for your changes to
     const event = this.parseEngineResponse<QueryEngineEvent | null>(log)
     if (!event) return
 
+    if (isSpanEvent(event)) {
+      return
+    }
```

Comment:
@danstarns Could you add a comment, maybe? Could help make this clearer

Reply:
I saw your comment that this is a temporary thing?

Reply:
This is temp yea!

Reply:
Probably ok to not add a comment then since you will change that in the feature branch right away